CODEOWNERS

```diff
@@ -172,18 +172,35 @@ Kconfig*                                  @tejlmand
 /modules/                                 @tejlmand
 /modules/tfm/                             @oyvindronningstad @ioannisg @hakonfam
 /subsys/zigbee/                           @tomchy @sebastiandraus
+/tests/                                   @gopiotr
 /tests/bluetooth/tester/                  @carlescufi @trond-snekvik
-/tests/drivers/flash_nop_device/          @de-nordic
 /tests/drivers/clock_control/clock_control_mpsl/ @ryanjh
-/tests/lib/edge_impulse/                  @pdunaj @MarekPieta
+/tests/drivers/flash_nop_device/          @de-nordic
+/tests/drivers/flash_patch/               @oyvindronningstad
+/tests/drivers/fprotect/                  @oyvindronningstad
+/tests/drivers/nrfx_integration_test/     @anangl
+/tests/lib/at_cmd_parser/                 @rlubos
 /tests/lib/at_sms_cert/                   @eivindj-nordic
+/tests/lib/date_time/                     @simensrostad
+/tests/lib/edge_impulse/                  @pdunaj @MarekPieta
 /tests/lib/hw_unique_key*/                @oyvindronningstad @Vge0rge
+/tests/lib/lte_lc/                        @jtguggedal
 /tests/lib/modem_jwt/                     @SeppoTakalo
+/tests/lib/sms/                           @trantanen
+/tests/modules/mcuboot/external_flash/    @hakonfam @sigvartmh
 /tests/modules/tfm/                       @hakonfam
+/tests/subsys/bluetooth/mesh/             @trond-snekvik
 /tests/subsys/bootloader/                 @hakonfam
-/tests/modules/mcuboot/external_flash/    @hakonfam @sigvartmh
+/tests/subsys/debug/cpu_load/             @nordic-krch
+/tests/subsys/dfu/                        @hakonfam @sigvartmh
+/tests/subsys/event_manager/              @pdunaj @MarekPieta
+/tests/subsys/fw_info/                    @oyvindronningstad
+/tests/subsys/net/lib/aws_jobs/           @simensrostad
+/tests/subsys/net/lib/aws_fota/           @simensrostad
```

Comment:
These two entries should be swapped so that the alphabetical order is kept. Or maybe `aws_*/` could be used instead?

Reply:
Yes, I agree. I chosed `aws_*/` option. Thanks for comment ;)